lib/cobrato/resources/payer.rb

```diff
@@ -1,7 +1,14 @@
 module Cobrato
   module Resources
     class Payer < Base
+      puts "Warning: Payers endpoint is deprecated. Please use People endpoint instead."
```

Comment:
I suggest to warn only when someone use the methods. See [regress_cnab.rb:9](https://github.com/myfreecomm/cobrato-client-ruby/blob/master/lib/cobrato/resources/regress_cnab.rb#L9)